### DISCUSSION TITLE
NumericUpDown floating point numbers

### DISCUSSION THREAD

**Discussion by JoachimEhbrecht** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3727] [Q&A] [answered]
Hello,
does the current NumericUpDown control (from 5.1.1) provides floating point numbers?
If so, how does it need to be configured (I think soming like an Increment-property is need)?
If not, is it planed to get this feature with the next release?

**Comment by nicolaihenriksen** [comment, marked as answer]
@JoachimEhbrecht In the coming release, the `NumericUpDown` will get a `decimal` counterpart named `DecimalUpDown`. I believe this is the control you're looking for. You should be able to test it by consuming one of the nightly builds (pre-release versions).

**Reply by JoachimEhbrecht** [reply]
Thank you for your reply.
I'm looking forward to the next versions.
